site/content/docs/main/self-signed-certificates.md

```diff
@@ -46,3 +46,12 @@ Error 116 represents certificate required as seen here in [error codes](https://
 Velero as a client does not include its certificate while performing SSL handshake with the server.
 From [TLS 1.3 spec](https://tools.ietf.org/html/rfc8446), verifying client certificate is optional on the server.
 You will need to change this setting on the server to make it work.
+
+
+## Skipping TLS verification
+
+**Note:** The `--insecure-skip-tls-verify` flag is insecure and susceptible to man-in-the-middle attacks and meant to help your testing and developing scenarios in an on-premise environment. Using this flag in production is not recommended.
+
+Velero provides a way for you to skip TLS verification on the object store by passing the `--insecure-skip-tls-verify` flag with Velero commands. If true, the object store's TLS certificate will not be checked for validity before Velero connects to the object store or Restic repo. You can permanently skip TLS verification for an object store by setting `Spec.Config.InsecureSkipTLSVerify` to true in the [BackupStorageLocation](api-types/backupstoragelocation.md) CRD.
```

Comment:
May need to be more specific with the Velero commands support parameter `--insecure-skip-tls-verify`.
By far, these commands supports the parameter:
* `velero backup describe`
* `velero backup download`
* `velero backup logs`
* `velero restore describe`
* `velero restore logs`

Reply:
Restic is integrated with Velero image. Velero run Restic commands to do data transfer between object store and k8s cluster disks.
Restic v0.13 supports a global command parameter `--insecure-skip` to skip TLS verification, and Velero is already updated to use this version of Restic. Velero AWS plugin and Velero Restic related controller watch for BSL's `Spec.Config.InsecureSkipTLSVerify` to determine whether to skip TLS verfication.

Reply:
Thanks for reviewing @blackpiglet, i've add some more changes that hopefully incorporate your commends correctly